packages/drpc/src/DrpcApi.ts

```diff
@@ -109,7 +110,7 @@ export class DrpcApi {
         removeListener: () => void
```

Comment:
Not related to this PR, but I'm not a big fan on reicvRequest over receiveRequest, that feels more consistent with other methods in Credo (we almost never abbreviate)